legacy-app/src/app/modules/events/pages/event-details-page/event-details-page.component.ts

```diff
@@ -138,7 +144,7 @@ export class EventDetailsPageComponent implements OnDestroy {
 
   async showCode() {
     const event = await firstValueFrom(this.event$);
-    if (event?.activeRegistration) {
+    if (event?.activeRegistration && !event.activeRegistration?.didAttend) {
```

Comment:
I think it would be better to hide the button or show a toast message explaining why nothing is happening

Reply:
Button is already hidden, this is just an extra step for the route

Reply:
ah okay, perfect then, will merge